utils/frame/remote-externalities/src/lib.rs

```diff
@@ -338,6 +338,17 @@ where
 			})
 	}
 
+	/// Returns the error message or suggestion for possible fixes based on the given RPC error.
+	fn get_batch_rpc_error_message(e: String) -> String {
+		// If the error contains "Invalid request ID", the request likely failed
+		// because the user incorrectly configured their node args
+		if e.to_string().contains("Invalid request ID") {
+			"Did you forget to specify `--rpc-max-request-size` or `--rpc-max-response-size` when starting your node? See https://paritytech.github.io/substrate/master/try_runtime_cli/index.html#note-on-nodes-that-respond-to-try-runtime-requests".to_string()
```

Comment:
That feels really hacky.

Reply:
Yeah it is. Do you have suggestions for paths I could explore to handle it better?

Reply:
yes… here https://github.com/paritytech/substrate/issues/13246  
Just re-try it with a smaller batch size until it works (and store that batch size for future use).